bin/src/prove.rs

```diff
@@ -78,6 +78,16 @@ fn main() {
     // Generating proofs for each trace
     let mut outer_timer = Measurer::new();
     for (trace_name, trace) in traces {
+        let tx_count = trace.transactions.len();
+        if tx_count > MAX_TXS {
+            panic!(
+                "{}",
+                format!(
+                    "Too many transactions. MAX_TXS:{}, given transactions:{}",
```

Comment:
```suggestion
                    "too many transactions. MAX_TXS:{}, given transactions:{}",
```
let's use lowercase for consistency